src/lib/components/ProductForm.svelte

```diff
@@ -126,7 +128,26 @@
 			} else {
 				priceAmountElement.setCustomValidity('');
 			}
+			let seen = new Set();
+			for (let i = 0; i < variationLabelsValues.length; i++) {
+				const key = `${(
+					variationLabelsNames[i] ||
+					product.variations?.[i].name ||
+					''
+				).toLowerCase()}-${(
+					variationLabelsValues[i] ||
+					product.variations?.[i].value ||
+					''
+				).toLowerCase()}`;
```

Comment:
Use `JSON.stringify({name, value})` - that way if I have a variation with name "a-b" and val "c", and another with "a" and "b-c", it won't cause issues